client/src/App.scss

```diff
@@ -121,6 +121,35 @@ Landing overview
   padding-right: 1rem;
 }
 
+.loginBtns {
+  display: flex;
+  width: 30%;
+  justify-content: space-around;
+  margin-bottom: 2 * $unit;
+}
+
+/*
+
+Modal
+
+*/
+
+.loginInput {
+  width: 75%;
+  margin: -5 * $unit 0 3 * $unit 3 * $unit;
+}
+
+input::placeholder {
+  line-height: 1.5;
+}
+
+.loginSubmitBtn {
+  position: relative;
+  width: 25%;
+  left: 70%;
+  bottom: 5%;
+}
```

Comment:
as mentioned in DM, this content should just by inside the modal body and then you can delete all these styles for positioning